sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/ApproximateQuantiles.java

```diff
@@ -358,6 +359,13 @@ private ApproximateQuantilesCombineFn(
         CoderRegistry registry, Coder<T> elementCoder) {
       return new QuantileStateCoder<>(compareFn, elementCoder);
     }
+
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("numQuantiles", numQuantiles)
+          .add("comparer", compareFn.getClass());
```

Comment:
I'm wondering if we want to also use the compareFn.toString(). I'm thinking about cases where a user may have constructed a comparator by wrapping one or more actual comparators. In that case, just knowing the class of the outer comparator may not be complete information. Also happy to defer until this becomes a problem.